stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/ScrollButtonViewStyle.kt

```diff
@@ -0,0 +1,77 @@
+package io.getstream.chat.android.ui.messages.view
+
+import android.content.res.TypedArray
+import android.graphics.drawable.Drawable
+import androidx.annotation.ColorInt
+import androidx.annotation.StyleableRes
+
+public class ScrollButtonViewStyle(
```

Comment:
`internal` constructors for these style classes?

Reply:
Sorry, on second thought, can't they be completely `internal`? Do we expose them anywhere?